webapp/assets/_new/styles/tokens.scss

```diff
@@ -15,7 +15,7 @@ $color-third: rgb(160,103,255);
 $color-success: rgb(23, 181, 63);
 $color-success-active: rgb(26, 203, 71);
 $color-success-inverse: rgb(241, 253, 244);
-$color-danger: rgb(219, 57, 36);
+$color-danger: rgb(25, 122, 49);
```

Comment:
You can not just change the main color for dangerous things @entermex .

Because this danger color is used in the code somewhere else and should stay red.